apps/web/app/(app)/onboarding/Steps.tsx

```
@@ -0,0 +1,81 @@
+  steps: {
+    title: string;
+    description: string;
+    content: React.ReactNode;
+    videoUrl?: string;
+    active: boolean;
+  }[];
+}) {
+  const router = useRouter();
+
+  return (
+    <ul role="list" className="space-y-6">
```

Biome

[error] Using the role attribute 'list' on the 'ul' element is redundant, because it is implied by the semantic 'ul' element.

Unsafe fix: Remove the role attribute.


(lint/a11y/noRedundantRoles)